src/modules/bot/commands/add-feed.command.ts

```diff
@@ -0,0 +1,34 @@
+import { InMemoryDBService } from '@nestjs-addons/in-memory-db'
+import { Logger } from '@nestjs/common'
+import { CommandBus, CommandHandler, ICommandHandler } from '@nestjs/cqrs'
+import { Message } from 'discord.js'
+import { PostFactorioCommand } from 'src/modules/tasks/commands/post-factorio.command'
+import { FeedEntity } from '../entities/feed.interface'
+
+export class AddFeedCommand {
+    constructor(public readonly content: string, public readonly context: Message) {}
+}
+
+@CommandHandler(AddFeedCommand)
+export class AddFeedCommandHandler implements ICommandHandler<AddFeedCommand> {
+    private readonly logger = new Logger(AddFeedCommandHandler.name)
+
+    constructor(private readonly feedService: InMemoryDBService<FeedEntity>, private readonly commandBus: CommandBus) {}
```

Comment:
We may need something better than InMemoryDBService. I wonder if I should run a nonsql instance in the VM. Is a text file db an option?

Reply:
anything is an option, though id like for something that has a nest-js lib so its just plug n play.